src/mixins/base-input.js

```diff
@@ -24,12 +24,15 @@ export default {
     },
     // called when input value change
     change() {
-      this.service.setEmpty();
+      this.service.setEmpty && this.service.setEmpty();
```

Comment:
@volterra79 this sounds weird, what are you trying to achieve exactly?

Reply:
@Raruto update